src/resources/product/controller.js

```diff
@@ -0,0 +1,13 @@
+const prisma = require("../../utils/database")
+
+const getAllProducts = async (req, res) => {
+    try {
+        const result = await prisma.product.findMany()
+        res.json([data = result])
```

Comment:
This doesn't look right to me. I think you're either looking for `res.json({ data: result })` or `res.json(result)`